__tests__/adaptiveCards.js

```diff
@@ -5,11 +5,8 @@ import { logging } from 'selenium-webdriver';
 import { imageSnapshotOptions, timeouts } from './constants.json';
 import allImagesLoaded from './setup/conditions/allImagesLoaded';
 import minNumActivitiesShown from './setup/conditions/minNumActivitiesShown';
-import scrollToBottomCompleted from './setup/conditions/scrollToBottomCompleted';
 import uiConnected from './setup/conditions/uiConnected';
 
-import createAdaptiveCardsHostConfig from '../packages/bundle/src/adaptiveCards/Styles/adaptiveCardHostConfig';
```

Comment:
Jest is (kinda) complaining that `createAdaptiveCardsHostConfig` is internally and should not be imported.